classes/PodsUI.php

```diff
@@ -4653,9 +4656,21 @@ public function pagination( $header = false ) {
 
 		if ( false !== $this->pagination ) {
 			if ( 1 < $total_pages ) {
+				$first_link    = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=1' );
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 4 spaces
* :no_entry_sign: Inline IF statements are not allowed

<sub>hash: e24010a1995caea163212a8024182cc7</sub>